aptos-move/framework/src/natives/aggregator_natives/aggregator.rs

```diff
@@ -62,6 +58,44 @@ fn native_add(
     Ok(smallvec![])
 }
 
+fn native_try_add(
+    gas_params: &AddGasParameters,
+    context: &mut SafeNativeContext,
+    _ty_args: Vec<Type>,
+    args: VecDeque<Value>,
+) -> SafeNativeResult<SmallVec<[Value; 1]>> {
+    match try_add(gas_params, context, _ty_args, args) {
```

Comment:
Something like this maybe `smallvec![Value::bool((try_add(gas_params, context, _ty_args, args).map(|_| true.unwrap_or(false))`?

Reply:
Simplified the match statement using `is_ok()` method.